packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -2676,14 +2676,9 @@ function commitMutationEffectsOnFiber(
               );
             }
           } else if (newResource === null && finishedWork.stateNode !== null) {
-            // We may have an update on a Hoistable element
-            const updatePayload: null | UpdatePayload =
-              (finishedWork.updateQueue: any);
-            finishedWork.updateQueue = null;
```

Comment:
Confirmed by logging in `yarn test` that `  finishedWork.updateQueu` is always `null` here